### PR TITLE
powdrvm keccak example

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: examples/keccak/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    let challenge: [u8; 32] = match Vec::from_hex(&args[1]) {
+        Ok(bytes) => {
+            if bytes.len() == 32 {
+                bytes.try_into().expect("length checked to be 32")
+            } else {
+                panic!("The pre-image must be exactly 64 hex characters (32 bytes).");
+            }
+        }
+        Err(e) => panic!("Error parsing the pre-image as hex: {e}"),
+    };
```
Comment (georgwiese) [inline examples/keccak/src/main.rs:25]: Maybe extract a `parse_hash` function?

**Review by georgwiese — COMMENTED** [review]
Looks good! I think I'll have to wait for #1980 to be merged to test it though.

File: examples/keccak/src/main.rs (hunk 1 above)
Comment (leonardoalt) [reply examples/keccak/src/main.rs:25]: u picky

Comment (leonardoalt) [reply examples/keccak/src/main.rs]: done

**Review by georgwiese — APPROVED** [review]
pwdrvm goooo

**Pull request merged**